staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5871,7 +5871,9 @@ type NodeDaemonEndpoints struct {
 	KubeletEndpoint DaemonEndpoint `json:"kubeletEndpoint,omitempty" protobuf:"bytes,1,opt,name=kubeletEndpoint"`
 }
 
-// NodeRuntimeHandlerFeatures is a set of runtime features.
+// NodeRuntimeHandlerFeatures is a set of features implementedy by the runtime handler.
+// THE FEATURES CONTAINED IN THE NodeRuntimeHandlerFeatures SHOULD DEPEND ON THE RUNTIME HANDLERS,
```

Comment:
I don't think we need to use ALL CAPS HERE.  Stylistically, we don't do that anywhere else

Reply:
fixed in https://github.com/kubernetes/kubernetes/pull/125470/commits/ca414b0f387753d8a6d4e77ec3563eb657e11f59